src/nanotron/parallel/pipeline_parallel/engine.py

```diff
@@ -2,6 +2,9 @@
 from typing import Dict, Iterable, Optional, Union
 
 import torch
+from torch import nn
+from torch.nn.parallel import DistributedDataParallel
```

Comment:
this is unrelated to this PR right? Im refactor this engine, I can take care of this change